pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -1950,12 +1951,36 @@ func ParseResults(result *resource.ResourceSearchResponse, offset int64) (*v0alp
 		Hits:      make([]v0alpha1.DashboardHit, len(result.Results.Rows)),
 	}
 
+	excludedFields := []string{
+		resource.SEARCH_FIELD_EXPLAIN,
+		resource.SEARCH_FIELD_SCORE,
+		resource.SEARCH_FIELD_TITLE,
+		resource.SEARCH_FIELD_FOLDER,
+		resource.SEARCH_FIELD_TAGS,
+	}
 	for i, row := range result.Results.Rows {
+		fields := &common.Unstructured{}
+		for colIndex, col := range result.Results.Columns {
+			if !slices.Contains(excludedFields, col.Name) {
```

Comment:
using a map here instead of slices.Contains would be better.  especially since this adds another nested loop.  this will be called a lot